<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QMainWindow" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Mode</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="moviesList"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="movieLabel">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>Movie</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addMovieButton">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateMovieButton">
            <property name="text">
             <string>update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteMovieButton">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="seeAllMoviesButton">
            <property name="text">
             <string>see all movies</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="titleLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="yearLabel">
            <property name="text">
             <string>Year of release:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="yearLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="genreLabel">
            <property name="text">
             <string>Genre:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="genreLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="likesLabel">
            <property name="text">
             <string>Number of likes:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="likeLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="trailerLabel">
            <property name="text">
             <string>Trailer:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="trailerLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="optionsLabel">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>Options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
